homeTask_5/homeTask_5/Book.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace homeTask_5
+{
+    class Book
+    {
+        public String Name { get; set; } = "";
+        public int PageCount { get; set; }
+        public String UniqueId { get; set; } = "";
+        public Book(String name, int pageCount, String uniqueId)
+        {
+            this.Name = name;
+            this.PageCount = pageCount;
+            this.UniqueId = uniqueId;
+        }
+        public Book()
+        {
+
+        }
+       
+        public void GetBook(Book book)
```

Comment:
Is it impossible to use properties in method instead of provide object to method in this object?

Reply:
> Is it impossible to use properties in method instead of provide object to method in this object?


If I understood you correctly, then I corrected everything